### PR TITLE
notify when a Needs Review label is added

### PR DESCRIPTION
**Pull request by ironcamel**

### CODE REVIEW

File: server.rb [hunk 1]
```diff
@@ -74,14 +74,17 @@ def comment_created_message(redis, repository, issue, comment)
   end
 end
 
-def pull_request_opened_message(redis, repository, pull_request)
-  score = redis.get(pull_request.user.login).to_i
+def handle_labeled_event(event, redis, slack)
+  repository = event.repository
+  issue = event.issue
+  return unless event.labeled?
```
Comment (cohitre) [inline server.rb]: move this out into the `post` block so that it follows the same pattern as the `comment_created?` flow.

File: server.rb [hunk 2]
```diff
@@ -74,14 +74,17 @@ def comment_created_message(redis, repository, issue, comment)
...
+  return unless repository.name =~ /site|internal-api|nightwatch|listener/;
```
Comment (cohitre) [inline server.rb]: if we want to start filtering for some repositories then we should filter all the events, not just the labeled ones

File: server.rb [hunk 3]
```diff
@@ -74,14 +74,17 @@ def comment_created_message(redis, repository, issue, comment)
   end
 end
 
-def pull_request_opened_message(redis, repository, pull_request)
-  score = redis.get(pull_request.user.login).to_i
+def handle_labeled_event(event, redis, slack)
+  repository = event.repository
+  issue = event.issue
+  return unless event.labeled?
+  return unless repository.name =~ /site|internal-api|nightwatch|listener/
```
Comment (cohitre) [inline server.rb]: if we want to start filtering for some repositories then we should filter all the events, not just the labeled ones

**Comment by cohitre** [comment]
if the notifications are getting too noisy then we should reevaluate our code review flow rather than having a whitelist for certain repos. 

we can turn off the webhooks for each repo from within github, that's a better solution than the whitelist.

**Comment by ironcamel** [comment]
I don't have admin access to github. We can remove the whitelist from here after we find someone to configure github for us. In the meantime, we need the whitelist to prevent spamming of the #eng channel.

**Comment by cohitre** [comment]
lgtm

**Pull request merged**